extensions/src/AWSSDK.Extensions.CloudFront.Signers/AWSSDK.Extensions.CloudFront.Signers.nuspec

```diff
@@ -5,10 +5,7 @@
     <title>AWSSDK - Extensions for AWS CloudFront</title>
     <version>4.0.1.0-preview</version>
     <authors>Amazon Web Services</authors>
-		<description>
-			This package contains extension methods for creating signed URLs for Amazon CloudFront distributions and
-			for creating signed cookies for Amazon CloudFront distributions using canned or custom policies.
```

Comment:
This line break was showing up in the Visual Studio UI:
![vs-ui](https://github.com/user-attachments/assets/59019e53-af4d-4406-ab6b-ba4ef00aee44)